compiler/src/dotty/tools/dotc/ast/Trees.scala

```diff
@@ -304,6 +304,7 @@ object Trees {
 
     def withFlags(flags: FlagSet): ThisTree[Untyped] = withMods(untpd.Modifiers(flags))
     def withAddedFlags(flags: FlagSet): ThisTree[Untyped] = withMods(rawMods | flags)
+    def withAddedAnnotation(annot: Tree[Untyped]): ThisTree[Untyped] = withMods(rawMods.withAddedAnnotation(annot))
```

Comment:
Dead code?

Reply:
Yes, not needed anymore. But I wanted to leave it in, since it's a useful complement to the other methods.